frontend/public/style/ancillary/_patternfly5.scss

```diff
@@ -9,68 +9,7 @@
   --pf-v5-global--FontSize--sm: 13px;
 }
 
-// Assigning PF component specific variables globally so we can use
-:root {
-  --pf-v6-c-code-block--BackgroundColor: var(--pf-v6-c-code-block--BackgroundColor);
-  --pf-v6-c-code-block__pre--FontFamily: var(--pf-v6-c-code-block__pre--FontFamily);
-  --pf-v6-c-code-block__pre--FontSize: var(--pf-t--global--font--size--sm);
-  --pf-v6-c-content--MarginBottom: var(--pf-t--global--spacer--md);
-}
-:where(:not([class*='pf-v6-c-'])) {
-  @at-root h1#{&} {
```

Comment:
is any of this CSS used by any dynamic plugin? we may want to defer removal until PF7, along with the rest of this file

Reply:
Good point.  I am of the opinion we should go ahead and force the breakage if there is any.  I really want console to get out of the business of supplying CSS such as this.

Reply:
You could add a note to the dynamic plugins README in "Changes in shared modules" that this styling is deprecated and will be removed in 4.20

Reply:
On third thought, I'm back to my first thought.  This rule will affect any non-PatternFly heading regardless of version.  Let's not do that.

Reply:
> You could add a note to the dynamic plugins README in "Changes in shared modules" that this styling is deprecated and will be removed in 4.20

We could, but we don't actually guarantee any CSS in the SDK.  So if this change breaks non-console code...

Reply:
> > You could add a note to the dynamic plugins README in "Changes in shared modules" that this styling is deprecated and will be removed in 4.20
> 
> We could, but we don't actually guarantee any CSS in the SDK. So if this change breaks non-console code...

I'm of the opinion that it still should be documented, just like how it documented that font awesome was removed from console, since it still affects plugins